documentation/source/users/rmg/input.rst

```diff
@@ -737,6 +737,8 @@ Miscellaneous options::
         saveSimulationProfiles=True,
         verboseComments=False,
         saveEdgeSpecies=True,
+        keepIrreversible=True,
+        trimolecularProductReversible=False,
```

Comment:
I don't think an option like this belongs in the options block.  It's a bit of an orphan option, but I think it probably best fits in either the database or generatedSpeciesConstraints blocks.

Reply:
I think this option is very similar in spirit to the `keepIrreversible` option, so I am inclined to leave it where it is.